src/Service.Host/client/src/components/CartonType.js

```diff
@@ -142,7 +122,6 @@ function CartonType({
                                 label="Description"
                                 fullWidth
                                 onChange={handleFieldChange}
-                                maxLength={50}
```

Comment:
Why did you need to remove these?

Reply:
hmmm.. I have no memory of removing that line. Will re-add before i get round to merging.